linkup-cli/src/main.rs

```diff
@@ -119,7 +119,8 @@ pub enum CliError {
 #[derive(Parser)]
 #[command(
     name = "linkup",
-    about = "Connect remote and local dev/preview environments"
+    about = "Connect remote and local dev/preview environments",
+    version = env!("CARGO_PKG_VERSION"),
```

Comment:
Noice! Don't think I had used [this](https://doc.rust-lang.org/std/macro.env.html) before.